### ISSUE TITLE
The default black background of VS Code is offensive.

### ISSUE THREAD

**Issue by wasddd-bit** [microsoft/vscode#87474]
The default black background of the editor is black. I am deply offended by that and I want you to change it. I think it is very offensive to all the black people due to the historical context. Chenge the default background to a different color and only make the black background available as an option that the user has to explicitly select.

**Comment by salifm** [comment]
I agree with you! Everything should be white! Change the default black background to white background!
